pylib/gyp/xcodeproj_file.py

```diff
@@ -651,7 +651,7 @@ def _XCPrintableValue(self, tabs, value, flatten_list=False):
         else:
             raise TypeError("Can't make " + value.__class__.__name__ + " printable")
 
-        if comment != None:
+        if comment is not None:
```

Comment:
```suggestion
        if comment:
```
No reason to add a space plus an empty string.